dbt_meshify/main.py

```diff
@@ -3,14 +3,46 @@
 
 from .dbt_projects import DbtProject, DbtSubProject, DbtProjectHolder
 
+# define common parameters
+project_path = click.option(
+    "--project-path",
+    default="."
+)
```

Comment:
I'd recommend that the project_path be typed as a Path, much like `project-dirs` below.

```suggestion
project_path = click.option(
    "--project-path",
    type=click.Path(exists=True),
    default="."
)
```